adm_program/modules/userrelations/relationtypes_function.php

```diff
@@ -67,9 +78,10 @@
         $relationtype2->setValue('urt_name', $_POST['urt_name_inverse']);
         $relationtype2->setValue('urt_name_male', empty($_POST['urt_name_male_inverse']) ? $_POST['urt_name_inverse'] : $_POST['urt_name_male_inverse']);
         $relationtype2->setValue('urt_name_female', empty($_POST['urt_name_female_inverse']) ? $_POST['urt_name_inverse'] : $_POST['urt_name_female_inverse']);
+        $relationtype2->setValue('urt_edit_user', $_POST['urt_edit_user_inverse']);
     }
 
-    // Daten in Datenbank schreiben
+    // write data into database
```

Comment:
I know. In my opinion it would be more proper to update step by step.
Not a mess, just to inform